src/eko/beta.py

```diff
@@ -7,7 +7,7 @@
 import numba as nb
 
 from . import constants
-from .harmonics.constants import zeta3
+from ekore.harmonics.constants import zeta3
```

Comment:
This one we said to merge into the global `constants` module, isn't it?

Reply:
I suggest to postpone to #202